blockchain/chain_sync/src/sync.rs

```diff
@@ -597,10 +610,9 @@ where
     }
 
     /// Validates block semantically according to https://github.com/filecoin-project/specs/blob/6ab401c0b92efb6420c6e198ec387cf56dc86057/validation.md
-    async fn validate(&self, block: &Block) -> Result<(), Error> {
+    async fn validate(&self, block: Block) -> Result<(), Error> {
```

Comment:
Reason for the switch from a reference? Cloning a block is not cheap at all, and it works fine to just use a reference

Reply:
Just remnants of dealing with a shared reference bugs, still cleaning up for sure :)